src/api/todos.ts

```diff
@@ -0,0 +1,20 @@
+import { Todo } from "../types/Todo";
+import { client } from "../utils/fetchClients";
+
+export const USER_ID = 1484;
+
+export function getTodos(USER_ID: number): Promise<Todo[]> {
```

Comment:
It is important to follow the same way of creating functions everywhere
```suggestion
export const getTodos = (USER_ID: number): Promise<Todo[]> => {
```